core/domain/config_services_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+from core.domain import config_domain
+from core.domain import config_services
+from core.tests import test_utils
+
+
+class ConfigServicesDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ConfigServicesDomainUnitTests, self).setUp()
+        self.committer_id = u'configServicesCommitterID'
+        self.invalid_property_name = u'_no_such_property_name'
+        self.property_dummy_value = u'TEST_VALUE'
+        self.schema_unicode = 'unicode'
+        self.config_schemas = (
+            config_domain.Registry.get_config_property_schemas())
+        # Get first property name, with unicode as schema type.
+        self.property_name = next(
+            x for x in self.config_schemas if
+            self.config_schemas[x]['schema']['type'] == self.schema_unicode)
+
+    def test_set_property(self):
+        with self.assertRaisesRegexp(Exception, (
+            'No config property with name')):
+            config_services.set_property(
+                self.committer_id, self.invalid_property_name,
+                self.property_dummy_value)
+        config_services.set_property(
```

Comment:
In general, when testing things, follow this format:

- test the "before" case
- do the action
- test the "after" case

So in this case you want to start by testing that the property isn't equal to property_dummy_value. And then after doing the action you want to test that it is equal to the expected value. Then you know that the action is working correctly.

Similar comments apply for the revert_property() case below, too.